### ISSUE TITLE
`primary_site` seems to be a duplicate of `composition` column

### ISSUE THREAD

**Issue by jaclyn-taroni** [AlexsLemonade/OpenPBTA-analysis#214]
#### File(s)

`pbta-histologies.tsv`

#### Release

`release-v7-20191031` but I think this is also a problem in `release-v6-20191030` but **not** in `release-v5-20190924`.

#### Link to [OpenPBTA-manuscript](https://github.com/AlexsLemonade/OpenPBTA-manuscript/)

N/A

#### Question/issue

The `primary_site` column in the `pbta-histologies.tsv` file appears to mirror the information in the `composition` column and no longer describes the "bodily site(s) from which specimen was derived" (per https://github.com/AlexsLemonade/OpenPBTA-manuscript/issues/49#issuecomment-548903626):

```R
> table(histologies_df$primary_site)

     Derived Cell Line           Not Reported Peripheral Whole Blood                 Saliva           Solid Tissue 
                    38                     12                    801                     18                   1964 
> table(histologies_df$composition)

     Derived Cell Line           Not Reported Peripheral Whole Blood                 Saliva           Solid Tissue 
                    38                     12                    801                     18                   1964 
```

I noticed it because it broke the code in `sample-distribution-analysis`: https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/213/files#diff-a712df3d5e6ad8758067c5d9ead79bb7

**Comment by jharenza** [comment]
Ahhhh, nice catch - coding error, but just fixed this. Will ask @yuankunzhu to update this file on s3 Monday so we can do a quick v8 release.

**Comment by jharenza** [comment]
fixed #223
